### PR TITLE
Final day 3 TLC

### PR DESCRIPTION
**Pull request by Avuxo**

### CODE REVIEW

File: script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_5_1.txt [hunk 1]
```diff
@@ -61,48 +61,48 @@ I mean, I feel dizzy like I always do.
...
 So there's no way it can be done with just a knife like this.
-In the first place, there is no such thing as a 'line', it's just a fantasy that I have in my mind.
+In the first place, there's no such thing as a 'line'; they're just a delusion that I came up with all on my own.//　あの『線』なんて初めからなくって、自分が勝手に思いこんでいるだけの妄想で――――
 <Page17>
 "――――It's a lie."
 The gastric juice keeps dripping from my lips.
-My mouth, from the chin down, is sticky with stomach fluids.
-The gastric juice blends with the color of red.
-There is nothing to vomit, but my stomach contracts, and my esophagus and my throat are bleeding.
+Ignoring my mouth, my body from the chin down is sticky with stomach fluids.//　口はおろか、あごから下は胃液でベトベトだ。
```
Comment (rschlaikjer) [inline script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_5_1.txt:72]: I think おろか is like a "**Not only** my mouth, but everything from the chin down..." meaning here? As in that it's a given that his mouth is sticky. "Ignoring" doesn't immediately parse like that to me, not sure about anyone else.

File: script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_6.txt [hunk 2]
```diff
@@ -57,42 +57,42 @@ It's true that my anemia is always a problem, but―――"
...
+"―――I'm sorry?"
 "I mean my clothes. My shirt, my shoes, there should have been blood―――"
 They were all wet and sticky.
 <Page17>
-"If you are talking about your uniform, it was dirty so I washed it."
+"If you are talking about your uniform, it was dirty and I washed it."
 "You're telling me you washed those bloody clothes...?!"
 <Page18>
-"...It is true that it was covered in mud, but there were none of the bloodstains that you speak of. Besides, it was only your shirt and pants that were dirty. Your jacket was not particularly dirty."
+"...It is true that it was covered in mud, but there were nothing like a bloodstain. Besides, it was only your shirt and pants that were dirty. Your jacket was not particularly dirty."
```
Comment (rschlaikjer) [inline script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_6.txt:67]: "There were nothing like a bloodstain" singular/plural mismatch

File: script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_6.txt [hunk 3]
```diff
@@ -57,42 +57,42 @@ It's true that my anemia is always a problem, but―――"
...
 <Page19>
 "――――――"
 ...Unbelievable.
-Even though I was kneeling in a pool of blood and my legs and arms were bright red...?
+Even though I was kneeling in a sea of blood and my legs and arms were bright red...?//　血の海にひざまずいて、足も腕も真っ赤になっていたっていうのに……？
 <Page20>
-"Shiki-sama, did you have a bad dream or something?
-You seemed like you had a terrible dream earlier, and your face is still pale even now."
+"Shiki-sama, did you have a some sort of bad dream?//「志貴さま、なにかよくない夢でも見られていたのですか？
```
Comment (rschlaikjer) [inline script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_6.txt:73]: "Have **~a~** some sort of bad dream"

File: script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_6.txt [hunk 4]
```diff
@@ -57,42 +57,42 @@ It's true that my anemia is always a problem, but―――"
...
+You seemed like you had a terrible dream earlier. Your face is still pale, even now."
```
Comment (rschlaikjer) [inline script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_6.txt:74]: _were having_ a terrible dream?
> 先ほどまで<魘|うな>されていたようですし、今もお顔の色が優れません」

File: script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_6.txt [hunk 5]
```diff
@@ -141,7 +141,7 @@ On the desk in the room is a knife, a memento from my father.
 It was nothing more than a dream.
 I close my eyelids, repeating the same phrase, as if to deceive myself.
 <Page37>
-...I let out a breath as if I'm trying to clear my lungs of the scent that I've taken in.
+...I breath in as if I'm trying to clear my lungs of that scent. A deep, deep breath.//　……肺に取りこんでしまった香気をかきだすように、長く長く息を吐く。
```
Comment (rschlaikjer) [inline script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_6.txt:144]: I breath**e** in

**Review by rschlaikjer — COMMENTED** [review]
Good stuff, just a couple editing artifacts :+1:

File: script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_5_1.txt [hunk 6]
```diff
@@ -69,7 +69,7 @@ In the first place, there's no such thing as a 'line'; they're just a delusion t
 <Page17>
 "――――It's a lie."
 The gastric juice keeps dripping from my lips.
-Ignoring my mouth, my body from the chin down is sticky with stomach fluids.//　口はおろか、あごから下は胃液でベトベトだ。
+Not only my mouth, my body from the chin down is sticky with stomach fluids.//　口はおろか、あごから下は胃液でベトベトだ。
```
Comment (rschlaikjer) [inline script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_5_1.txt:72]: feels like you need a `but` after the comma?
Not only my mouth, **but my entire body** from the chin down is sticky with stomach fluids.

File: script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_6.txt [hunk 7]
```diff
@@ -57,42 +57,42 @@ It's true that my anemia is always a problem, but―――"
...
+That feeling?
+The smell of blood?
+That ethereal woman in white?
+"Yeah―――you're right. That must have all been a bad dream."
 <Page22>
 I let out a long sigh.
 ――――That's right.
 It was all a bad dream.
-It's not like I was going to break a promise I made to Sensei when I was little and do something like that for no reason.
+It's not like I'd do something like that without meaning or reason and break a promise I made with Sensei when I was little.//　意味もなく理由もなく、幼い頃に先生と交わした約束を破って、俺があんなコトをするわけがなかったんだ。
```
Comment (rschlaikjer) [inline script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_6.txt:87]: promise I made **to** is correct I think
2021-12-23-174442_1437x491_scrot

Comment (Avuxo) [reply script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_6.txt:87]: "promise with" is still grammatically correct and is a more accurate translation of "先生と交わした約束" imo.

**Review by rschlaikjer — APPROVED** [review]

**Review by yk1512 — DISMISSED** [review]

**Pull request merged**